pages/content/amp-dev/documentation/guides-and-tutorials/integrate/story-interactive.md

```diff
@@ -0,0 +1,71 @@
+formats:
+  - stories 
+---
+
+Interactive experiences provide a first-class support for user interactivity in Web Stories, and are designed with customization in mind. However, all the customization options can be overwhelming for users designing stories with interactive elements. We will be covering ways tools can integrate the interactive experiences into their existing workflows, and best practices to take into account.
+
+## Import scripts
+
+Start by importing the script in the head of your document:
+
+```html
+<script async custom-element="amp-story-interactive" src="https://cdn.ampproject.org/v0/amp-story-interactive-0.1.js"></script>
+```
+
+## Understanding the elements
+
+Understanding how the elements work is crucial to integrating them successfully, so we have created [a Codepen collection](https://codepen.io/collection/DEGRLE) to host demos of the components and some of their styling variants.
+
+Check out our guides on [how to integrate polls](https://amp.dev/documentation/examples/components/amp-story-interactive-poll/?format=stories) or [how to integrate quizzes with analytics](https://amp.dev/documentation/examples/components/amp-story-interactive-quiz/?format=stories) to Web Stories.
+
+## Tips on integrating into tools
+
+We have provided [Codepens](https://codepen.io/collection/DEGRLE) to show how the interactive elements might look in an editor. As shown there, it's possible to offer inline editing of the content, by reusing the HTML and CSS from each demo. Individual elements such as quizzes or polls have limited content customization besides the prompt and options content, but can be customized using many combinations of CSS variables and attributes.
+
+### Sizing
+
+The interactive elements work great across different screen resolutions by having a size that is dependent on the `font-size` value, which is by default set to a percentage of the screen size to adapt the elements correctly. However, story creators can resize the elements by changing the `font-size` rule. The recommended way of integrating sizing into editors is either to not allow users to change the size at all, or to give users a slider with good boundaries. For instance, if a tool wants to use `amp-story-grid-layer[aspect-ratio]` layouts, they might want to set a slider to go between 0.15ems and 0.3ems, which are good extremes for very large and very small elements. Tools can use ems, rems or percentage of `var(--story-page-vmin)` for instance, but is recommended to use the same type of units that they use for text, as these elements can be thought of as "regular text containers". Changing the `width` property should only be made within the predefined min and max values (if at all).
```

Comment:
Can we just say that the components height will automatically adapt to the size of its text content so all it takes is setting a font-size, + what you have about the width?
This section is so so so confusing

Edit: this is actually already documented somewhere else this is not a tip about creation tool integration